tests/test_database_types.py

```diff
@@ -24,7 +31,7 @@
         "2022-05-01 15:10:03.003030",
         "2022-06-01 15:10:05.009900",
     ],
-    "float": [0.0, 0.1, 0.10, 10.0, 100.98],
+    "float": [0.0, 0.1, 0.00188, 0.99999, 0.091919, 0.10, 10.0, 100.98],
```

Comment:
Some additional values that cause some more tests to fail:

```
     "float": [0.0, 0.1, 0.00188, 0.99999, 0.091919, 0.10, 10.0, 100.98,
         0.001201923076923077, 1/3, 1/5, 1/109, 1/109489, 1/1094893892389, 1/10948938923893289],
```

Failing tests:

* test_types_Snowflake_float_to_Postgres_real
* test_types_Redshift_float8_to_Postgres_real
* test_types_Postgres_real_to_Snowflake_float
* test_types_Postgres_real_to_Redshift_float8
* test_types_Postgres_real_to_Postgres_double_precision
* test_types_Postgres_real_to_MySQL_numeric_65_10_
* test_types_Postgres_double_precision_to_Postgres_real
* test_types_MySQL_numeric_65_10__to_Postgres_real

Reply:
Nice. I'll look into it!

Reply:
Working now.